src/live_restore/live_restore_server.c

```diff
@@ -362,17 +379,19 @@ __wt_live_restore_server_create(WT_SESSION_IMPL *session, const char *cfg[])
         return (0);
 
     WT_CONNECTION_IMPL *conn = S2C(session);
-    WT_ERR(__wt_calloc_one(session, &conn->live_restore_server));
 
     /* Read the threads_max config, zero threads is valid in which case we don't do anything. */
     WT_CONFIG_ITEM cval;
     WT_ERR(__wt_config_gets(session, cfg, "live_restore.threads_max", &cval));
     if (cval.val == 0)
         return (0);
 
+    WT_ERR(__wt_calloc_one(session, &conn->live_restore_server));
     WT_ERR(__wt_spin_init(
       session, &conn->live_restore_server->queue_lock, "live restore migration work queue"));
 
+    __wt_atomic_storebool(&conn->live_restore_server->shutting_down, false);
```

Comment:
Is this so there's obvious pairing? Clearly the calloc on the line above achieves this for us.

Reply:
Yep. calloc does the job but I want to keep things explicit.
I decided not to wrap this in the queue lock since we're in the single threaded setup stage. I can add the lock if you want/if we think TSan will complain.